common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON

```diff
@@ -0,0 +1,1256 @@
+								  },
+								  {
+									  "key": "Content-Type",
+									  "value": "application/json",
+									  "type": "text"
+								  }
+							  ],
+							  "body": {
+								  "mode": "raw",
+								  "raw": "{\"name\": \"<gwConfigurations_name>\", \"base64_sic_key\": \"<encoded_base64_sic_key>\", \"version\": \"<gw_version>\", \"policy\": \"<policy_name>\", \"blades\": {\"ips\":true, \"anti-bot\":true, \"url-filtering\": true, \"https-inspection\": true}, \"related_account\": \"<associated_account>\"}"
+							  },
+							  "url": {
+								  "raw": "{{server}}/v1.8/cme-api/v1.2/gwConfigurations/azure",
+								  "host": [
+									  "{{server}}"
+								  ],
+								  "path": [
+									  "v1.8",
+									  "cme-api",
+									  "v1.2",
+									  "gwConfigurations",
+									  "azure"
+								  ]
+							  }
+						  },
+						  "response": []
+					  },
+					  {
+						  "name": "PUT set Azure gw configuration",
+						  "request": {
+							  "method": "PUT",
+							  "header": [
+								  {
+									  "key": "X-chkp-sid",
+									  "value": "{{session}}",
+									  "type": "text"
+								  },
+								  {
+									  "key": "Content-Type",
+									  "value": "application/json",
+									  "type": "text"
+								  }
+							  ],
+							  "body": {
+								  "mode": "raw",
+								  "raw": "{\"base64_sic_key\": \"<encoded_base64_sic_key>\", \"version\": \"<gw_version>\", \"policy\": \"<policy_name>\", \"blades\": {\"ips\":true, \"anti-bot\":true, \"url-filtering\": true, \"https-inspection\": true}, \"related_account\": \"<associated_account>\"}"
+							  },
+							  "url": {
+								  "raw": "{{server}}/v1.8/cme-api/v1.2/gwConfigurations/azure/<gwConfigurations_name>",
+								  "host": [
+									  "{{server}}"
+								  ],
+								  "path": [
+									  "v1.8",
+									  "cme-api",
+									  "v1.2",
+									  "gwConfigurations",
+									  "azure",
+									  "<gwConfigurations_name>"
+								  ]
+							  }
+						  },
+						  "response": []
+					  }
+					]
+		},
+		{
+			"name": "AWS",
+			"description" : "Operations in AWS cloud.",
+			"item": [
+				{
+					"name": "POST add AWS account",
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"value": "application/json"
+							},
+							{
+								"key": "X-chkp-sid",
+								"value": "{{session}}"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\"name\": \"<account_name>\", \"regions\": [\"eu-west-1\", \"us-east-2\"], \"credentials_file\": \"AWS_credentials_file_name\", \"access_key\": \"<access_key>\", \"secret_key\": \"<secret_key>\", \"sts_role\": \"<sts_role>\", \"sta_external_id\": \"<sts_external_id>\", \"scan_gateways\": \"<scan_gateways>\", \"scan_vpn\": \"<scan_vpn>\", \"scan_load_balancers\": \"<scan_load_balancers>\", \"access_subnets\": \"<access_sunbets>\", \"deletion_tolerance\": \"<deletion_tolerance>\"}",

```

Comment:
Please change to "raw": "{\"name\": \"<account_name>\", \"regions\": [\"eu-west-1\", \"us-east-2\"], \"credentials_file\": \"AWS_credentials_file_name\", \"access_key\": \"<access_key>\", \"secret_key\": \"<secret_key>\", \"sts_role\": \"<sts_role>\", \"sts_external_id\": \"<sts_external_id>\", \"scan_gateways\": \"<scan_gateways>\", \"scan_vpn\": \"<scan_vpn>\", \"scan_load_balancers\": \"<scan_load_balancers>\", \"scan_subnets\": \"<scan_subnets>\", \"communities\": [\"<community_a>\", \"<community_b>\"], \"deletion_tolerance\": \"<deletion_tolerance>\", \"sub_accounts\": [{\"name\": \"sub_account_a\",\"credentials_file\": \"AWS_credentials_file_name\"}, {\"name\": \"sub_account_b\", \"access_key\": \"<access_key>\", \"secret_key\": \"<secret_key>\"}, {\"name\": \"sub_account_c\", \"sts_role\": \"<sts_role>\", \"sts_external_id\": \"<sts_external_id>\"}]}"